code/ship/shipfx.cpp

```diff
@@ -2474,7 +2474,7 @@ void shipfx_do_lightning_frame( ship * /*shipp*/ )
 	shipp->lightning_stamp = timestamp(stamp);
 
 	// ah, now we can create some lightning bolts
-	count = (int)frand_range(0.0f, (float)count);
+	count = Random::next(0, count);
```

Comment:
Looks like some of these can be just `Random::next(max)` now? Trivial, but nice to be consistent

Reply:
Some, but not all.  `Random::next(low, high)` will include both the low and the high value, while `Random::next(modulus)` will include 0 but exclude modulus.  Here, the full count is a valid possibility so the (0, N) call should be used.

Reply:
Ah, I see. Perhaps `Random::next(max+1)`? It's not a big deal, but it's not super obvious just by looking at it.

Also, on second thought, I believe float casting rounds down, so a frand_range will basically never include its top value anyway.

Reply:
Ok, how's this?

Reply:
Now, I'm starting to have second thoughts lol so I'm just going to approve it before I bikeshed it further and it really isn't a big deal